game/kernel/kprint.cpp

```diff
@@ -884,6 +884,8 @@ s32 format_impl(uint64_t* args) {
                 call_method_of_type(in, type, GOAL_PRINT_METHOD);
               }
             } else {
+              // TODO - if we can't throw exceptions, what is the option?
+              // log, break and continue?
               throw std::runtime_error("failed to find symbol in format!");
```

Comment:
wasn't sure about how to fix these

Reply:
I think it's fine for now. I'm planning to add some better handling in the runtime for GOAL crashes that will include an `abort` which can safely be called in C code called from GOAL.